cpp/include/IceSSL/OpenSSL.h

```diff
@@ -59,7 +21,7 @@ namespace IceSSL
          * Encapsulates an OpenSSL X.509 certificate.
          * \headerfile IceSSL/IceSSL.h
          */
-        class ICESSL_OPENSSL_API Certificate : public virtual IceSSL::Certificate
+        class ICE_API Certificate : public virtual IceSSL::Certificate
```

Comment:
Can we remove the virtual, or do we actually use multiple inheritance with Certificates?

Reply:
Yes I think we can. We should first consider if we want to keep them.

The main use case I see now for this class is to inspect the peer certificate in the connection info.